### DISCUSSION TITLE
How can I  make scrollable form section

### DISCUSSION THREAD

**Discussion by tahinuralam** [filamentphp/filament#12341] [Help]
### Package

Form builder

### Package Version

3.2.66

### How can we help you?

Split::make([
    Section::make([
        TextEntry::make('title')
            ->weight(FontWeight::Bold),
        TextEntry::make('content')
            ->markdown()
            ->prose(),
    ]),
    Section::make([
        TextEntry::make('created_at')
            ->dateTime(),
        TextEntry::make('published_at')
            ->dateTime(),
    ])->grow(false),
])->from('md')

**Comment by leandrocfe** [comment]
What is the issue?

**Reply by tahinuralam** [reply]
![Screenshot_20240419_160500](https://github.com/filamentphp/filament/assets/48985252/cec1333a-d4fc-493d-9a33-37041744d828)

**Reply by tahinuralam** [reply]
The created at  section will be fixed when the title section is scrolling

**Comment by shreejanpandit** [comment]
I guess this helps !

`use Filament\Forms\Components\Section;
use Filament\Forms\Components\Group;

Section::make('My Section')
    ->schema([
        Group::make([
            // Your form fields here
            TextInput::make('name'),
            Select::make('country')
                ->options(['us' => 'USA', 'ca' => 'Canada']),
            // Add more fields...
        ])
        ->extraAttributes([
            'style' => 'max-height: 400px; overflow-y: auto; padding: 0 8px;'
        ])
        ->columnSpanFull()
    ])
    ->columns(2)`
